Xamarin.Forms.Xaml.UnitTests/SetValue.xaml.cs

```diff
@@ -18,6 +18,38 @@ public class MockViewWithValues : View
 	{ 
 		public UInt16 UShort { get; set; }
 		public decimal ADecimal { get; set; }
+		public SV_Foo Foo { get; set; }
+		public string Bar { get; set; }
+
+		public static readonly BindableProperty BPFooProperty =
+			BindableProperty.Create("BPFoo", typeof(SV_Foo), typeof(MockViewWithValues), default(SV_Foo));
+
+		public SV_Foo BPFoo {
+			get { throw new NotImplementedException(); }
+			set { throw new NotImplementedException(); }
+		}
```

Comment:
throw NRE to make sure the values are set using SetValue and not the property accessor. We still have to declare the property so XamlC knows the type of the BP